Documentation/GettingStarted.md

```diff
@@ -16,27 +16,27 @@ By default this will show a screen similar to the one below:
 
 ![Home Screen](/Documentation/images/HomeScreen.png)
 
-Note that this may be drawn mirrored depending on the orientation of your screen.
+Note that this may be drawn mirrored depending on the orientation of your screen (detailed mode shows a different home screen).
 
 The soldering iron symbol on the screen will appear near the tip. This is here to indicate that pressing the button closest to the front of the iron will enter soldering mode.
 
 And naturally, the spanner like icon represents that pressing the button near the rear of the soldering iron will enter the settings menu.
 
-In the settings, you can turn on an advanced idle screen instead. The buttons still function the same, however, the image will be swapped for a text telling you the current status of the iron with extra details.
+In the settings, you can turn on a detailed idle screen instead. The buttons still function the same, however, the image will be swapped for a text telling you the current status of the iron with extra details.
 
-Depending on how your device is being powered, at one end of the screen, the firmware will either draw text to show you the current-voltage your unit is being provided with, a battery icon (if battery mode is enabled) or a power plug icon.
+Depending on how your device is being powered, at right side of the screen, the firmware will either show the voltage your unit is being provided with, a battery icon (if battery mode is enabled) or a power plug icon.
 
-If you see exclamation marks (!!!) where the soldering iron should be, this indicates that the firmware can't see the tip connected. This could indicate a problem with the iron or tip if there is one inserted. First, try removing and reinstalling the tip. If the issue persists please see the [hardware issues section](/Documentation/HardwareIssues.md).
+If you see an (**X**) where the soldering iron should be, this indicates that the firmware can't see the tip connected. This could indicate a problem with the iron or tip. First, try removing the tip screw and tip and gently reinstalling both; ensure that the tip is seated all the way back. If the issue persists please see the [hardware issues section](/Documentation/HardwareIssues.md).
```

Comment:
```suggestion
If you see a crossed out tip instead of only a tip, this indicates that the firmware can't see the tip connected. This could indicate a problem with the iron or tip if there is one inserted. First, try removing and reinstalling the tip; ensure that the tip is seated all the way back. If the issue persists please see the [hardware issues section](/Documentation/HardwareIssues.md).
```
The sentence does not make much sense, as long as you remove the ***'if part'*** of that sentence.
How can your device know if your tip has a problem, if the tip is not inserted.

Not every device supporting IronOS uses a screw to fasten the tip.